zerver/views/presence.py

```diff
@@ -46,6 +51,25 @@ def get_presence_backend(request: HttpRequest, user_profile: UserProfile,
         val.pop('pushable', None)
     return json_success(result)
 
+@human_users_only
+@has_request_variables
+def update_user_status_backend(request: HttpRequest,
+                               user_profile: UserProfile,
+                               away: bool=REQ(validator=check_bool, default=False),
```

Comment:
Do we want a default on this?  I think it'd be better for the parameter to either be required, or to allow `None` and be a no-op in the case (for consistency when one might want to be only changing other properties of the user's status, like a message).

Reply:
oops, that was cargo culted...we don't want a default there for now, and I'll add a little negative test